electron_app/src/backup/databaseExport.js

```diff
@@ -0,0 +1,466 @@
+const fs = require('fs');
+const moment = require('moment');
+const {
+  AccountContact,
+  Alias,
+  Contact,
+  CustomDomain,
+  Email,
+  Label,
+  getDB,
+  Table,
+  getSettings
+} = require('../database/DBEmanager');
+const systemLabels = require('./../systemLabels');
+const { APP_DOMAIN, LINK_DEVICES_FILE_VERSION } = require('../utils/const');
+const { getEmailBody, getEmailHeaders } = require('../utils/FileUtils');
+
+const excludedEmailStatus = [1, 4];
+const excludedLabels = [systemLabels.draft.id];
+const whereRawEmailQuery = (id, accountId) => `INNER JOIN ${Table.EMAIL_LABEL} 
+    ON ${id} = ${Table.EMAIL_LABEL}.emailId 
+    WHERE ${Table.EMAIL}.accountId=${accountId} 
+    AND ${Table.EMAIL_LABEL}.labelId NOT IN (${excludedLabels.join(',')}) 
+    AND ${Table.EMAIL}.status NOT IN (${excludedEmailStatus.join(',')})
+`;
+
+/* Batches
+----------------------------- */
+const SELECT_ALL_BATCH = 500;
```

Comment:
Are you sure to set the same batch for all the tables? The current value for emails is 100.

Reply:
Yes, the fewer queries the better I believe